android/src/main/java/com/dooboolab/flutterinapppurchase/AmazonInappPurchasePlugin.java

```diff
@@ -4,32 +4,28 @@
 import android.content.Context;
```

Comment:
@hyochan What is the status on amazon IAP? Is it a working feature? I tried to unify Amazon IAPs maps to be the same as Android apps, but it is better to kill amazon IAPs in case it is not a working feature and nobody uses it.

Reply:
@bohdan1krokhmaliuk Honestly I've not tried Amazon at all because I am not using it. Also, this was developed by the community by @Rockvole.

How about separating the two variants as done in https://github.com/dooboolab/react-native-iap/pull/1336?

Reply:
No no, I was wondering cause it would make two more variables not nullable

Reply:
Hi, Sorry I cant help much, I have not done any Android java development since this Amazon IAP work (2018 I think). The Amazon store may become more important soon since it will be used for Android apps in Windows 11. 
https://www.theverge.com/2021/6/24/22548428/microsoft-windows-11-android-apps-support-amazon-store